b2sdk/api.py

```diff
@@ -263,6 +263,36 @@ def download_file_by_id(
             encryption,
         )
 
+    def update_file_retention(
+        self,
+        file_id: str,
+        file_name: str,
+        file_retention: FileRetentionSetting,
+        bypass_governance: bool = False,
+    ):
+        return FileIdNameAndRetention.from_response(
```

Comment:
`FileIdNameAndRetention` class feels really artificial - user had to pass all data that it contains - and errors are reported using exceptions - I would just return nothing